### PR TITLE
new(Icons): Add IconCode

### PR DESCRIPTION
**Pull request by al-chen**
to: @milesj @stefhatcher @hayes @alecklandgraf
cc: @tyn2

## Description

Add the code icon from https://material.io/tools/icons/?search=code&icon=code&style=baseline

## Motivation and Context

Used for Guide Designer UI

## Testing

verified in storybook

## Screenshots

Screen Shot 2019-06-11 at 2 13 50 PM

## Checklist

- My code follows the style guide of this project.
- I have updated or added documentation accordingly.
- I have read the CONTRIBUTING document.

### CODE REVIEW

File: svgs/general/code.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"><path fill="none" d="M0 0h24v24H0V0z"/><path d="M9.4 16.6L4.8 12l4.6-4.6L8 6l-6 6 6 6 1.4-1.4zm5.2 0l4.6-4.6-4.6-4.6L16 6l6 6-6 6-1.4-1.4z"/></svg>
```
Comment (milesj) [inline svgs/general/code.svg]: Can you remove `fill="none"`.

File: svgs/general/code.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"><path d="M0 0h24v24H0V0z"/><path d="M9.4 16.6L4.8 12l4.6-4.6L8 6l-6 6 6 6 1.4-1.4zm5.2 0l4.6-4.6-4.6-4.6L16 6l6 6-6 6-1.4-1.4z"/></svg>
```
Comment (stefhatcher) [inline svgs/general/code.svg]: this is showing up as a black box.
Screen Shot 2019-06-12 at 9 43 12 AM

when you click the paper icon:
Screen Shot 2019-06-12 at 9 43 57 AM


if you remove `<path d="M0 0h24v24H0V0z"/>`, it is fixed. it was the path with `fill="none"` before and doesn't look like it is part of the main svg path.

Comment (al-chen) [reply svgs/general/code.svg]: Thanks! I've corrected and verified in local storybook, it should now work per your suggestion.

Comment (stefhatcher) [reply svgs/general/code.svg]: hrmm did you push the changes? still seeing that unused path in the source code and generated component

update the base svg from:
```svg
<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24">
  <path fill="none" d="M0 0h24v24H0V0z"/>
  <path d="M9.4 16.6L4.8 12l4.6-4.6L8 6l-6 6 6 6 1.4-1.4zm5.2 0l4.6-4.6-4.6-4.6L16 6l6 6-6 6-1.4-1.4z"/>
</svg>
```

to:
```svg
<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24">
  <path d="M9.4 16.6L4.8 12l4.6-4.6L8 6l-6 6 6 6 1.4-1.4zm5.2 0l4.6-4.6-4.6-4.6L16 6l6 6-6 6-1.4-1.4z"/>
</svg>
```

File: svgs/general/code.svg (hunk 1 above)
Comment (stefhatcher) [inline svgs/general/code.svg]: remove `<path fill="none" d="M0 0h24v24H0V0z"/>` from this source and re-generate the icon component (or just remove that same line from the generated IconCode)

Comment (al-chen) [reply svgs/general/code.svg]: @stefhatcher there it is...think the commit somehow lost when I tried rebasing

Comment (stefhatcher) [reply svgs/general/code.svg]: aah! i see it now! thank you!

**Pull request merged**